Trimble.Modus.Components/Controls/ListView/TextCell.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ViewCell xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters"
+             xmlns:constant="clr-namespace:Trimble.Modus.Components.Constant"
+             xmlns:helpers="clr-namespace:Trimble.Modus.Components.Helpers"
+             x:Name="textCell"
+             x:Class="Trimble.Modus.Components.TextCell">
+  <Grid x:Name="grid" >
+    <Grid.Resources>
+      <ResourceDictionary>
+        <converters:ImageNullOrEmptyBoolConverter x:Key="ImageNullOrEmptyBoolConverter" />
+        <converters:StringNotNullOrEmptyBoolConverter x:Key="StringNotNullOrEmptyBoolConverter" />
+        <converters:DescriptionToRowHeightConverter x:Key="DescriptionToRowHeightConverter" />
+        </ResourceDictionary>
+    </Grid.Resources>
+
+    <Grid BindingContext="{Binding Source={x:Reference textCell}}" HeightRequest="{Binding Description, Converter={StaticResource DescriptionToRowHeightConverter}}">
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="Auto" />
+        <ColumnDefinition Width="*" />
+        <ColumnDefinition Width="Auto" />
+      </Grid.ColumnDefinitions>
+
+      <Image Source="{Binding LeftIconSource}" IsVisible="{Binding LeftIconSource, Converter={StaticResource ImageNullOrEmptyBoolConverter}}"
+           WidthRequest="24" HeightRequest="24" VerticalOptions="Center" HorizontalOptions="Start" Margin="12,0,0,0" />
```

Comment:
Instead of adding margin in all controls we can add columnspacing as 12

Reply:
Added column spacing and padding to grid removed padding and margin used inside the content